src/main/java/uk/gov/hmcts/reform/bulkscanccdeventhandler/caseupdate/model/in/CaseUpdateDetailsRequest.java

```diff
@@ -0,0 +1,50 @@
+package uk.gov.hmcts.reform.bulkscanccdeventhandler.caseupdate.model.in;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import uk.gov.hmcts.reform.bulkscanccdeventhandler.common.model.in.InputScannedDoc;
+import uk.gov.hmcts.reform.bulkscanccdeventhandler.common.model.in.JourneyClassification;
+import uk.gov.hmcts.reform.bulkscanccdeventhandler.ocrvalidation.model.in.OcrDataField;
+
+import java.time.LocalDateTime;
+import java.util.List;
+
+public class CaseUpdateDetailsRequest {
```

Comment:
Why does this class have `Request` suffix?

Reply:
There is already a class called CaseUpdateDetails.

Reply:
Renamed CaseUpdateDetailsRequest to CaseUpdateDetails though I do not like these class name clashes.